src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -437,7 +441,6 @@ bool CoreWorkerMemoryStore::Contains(const ObjectID &object_id, bool *in_plasma)
   if (it != objects_.end()) {
     if (it->second->IsInPlasmaError()) {
       *in_plasma = true;
-      return false;
```

Comment:
Is it okay to remove this?

Reply:
Yep! Just made the API a bit more sane (updated header and callers)